config/data_sources/covid_tracking_project.tf

```diff
@@ -5,3 +5,20 @@ resource "google_bigquery_dataset" "bq_covid_tracking_project" {
   dataset_id = "covid_tracking_project"
   location   = "US"
 }
+
+# We need to define the metadata table in Terraform so that we can grant
```

Comment:
Isn't it generally recommended to define all resources in tf? It seems like this comment should go with the binding bellow.

Reply:
Whoops, somehow this response was created in its own thread: 

IIUC we don't actually define any other BQ tables, just the datasets. Our other tables are created when writing to BQ or through BQ routines.